todo_app/app.py

```diff
@@ -1,15 +1,22 @@
-from flask import Flask
+from flask import Flask, render_template, redirect, url_for
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items, add_item
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
-@app.route('/')
+@app.route('/', methods=['GET','POST'])
```

Comment:
This endpoint should only need to return on a GET request, so we could remove the POST method option here